src/hooks/useVenueDetails.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+
+export const useVenueDetails = (id) => {
+  const [venue, setVenue] = useState(null);
+  const [loading, setLoading] = useState(true);
```

Comment:
I suggest prefixing most booleans with `is` for clarity. So, for example, `isLoading` instead of `loading`